### PR TITLE
Use bundle to execute commands

### PR DESCRIPTION
**Pull request by dmydlarz**
Add flag to make it possible to run build commands via bundle.

Additionally add build.sh script. I spend some time figuring out how to build & install locally.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this pull request

File: bash/deploy.sh [hunk 1]
```diff
@@ -22,10 +23,18 @@ cd "${CLONE}"
 echo -e "\nBuilding Jekyll site:"
 rm -rf _site
 
-if [ -r ${DEPLOY_CONFIG} ]; then
-  jekyll build --config _config.yml,${DEPLOY_CONFIG}
+if [ -r ${BUNDLE} ]; then
+  if [ -r ${DEPLOY_CONFIG} ]; then
```
Comment (yegor256) [inline bash/deploy.sh]: @dmydlarz looks like code duplication for me. How about passing `$5` as `bundle exec` and just using it here.

File: bin/jgd [hunk 2]
```diff
@@ -22,6 +23,7 @@ fail 'branch-from can\'t be empty' if branch_from.empty?
 fail 'config can\'t be empty' if config.empty?
 url = opts[:url]
 url = `git config --get remote.origin.url`.strip if url.empty?
+bundle = opts[:bundle]
```
Comment (yegor256) [inline bin/jgd]: @dmydlarz here: `bundle = opts[:bundle] ? 'bundle exec' : ''`

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (yegor256) [inline build.sh]: @dmydlarz maybe let's just put this into README? What's the point of having this file here?

**Comment by yegor256** [comment]
@dmydlarz see my comments above

**Comment by dmydlarz** [comment]
@yegor256 what do you think now?

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmydlarz** [comment]
@yegor256 I am not into RubyGems at all. Is it gonna be published there somehow automatically?

**Comment by yegor256** [comment]
@rultor release, tag is `1.10`

**Pull request merged**